src/qt/utilitydialog.cpp

```diff
@@ -100,6 +100,7 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, HelpMode helpMode) :
         strUsage += HelpMessageOpt("-resetguisettings", tr("Reset all settings changed in the GUI").toStdString());
         if (showDebug) {
             strUsage += HelpMessageOpt("-uiplatform", strprintf("Select platform to customize UI for (one of windows, macosx, other; default: %s)", BitcoinGUI::DEFAULT_UIPLATFORM));
+            strUsage += HelpMessageOpt("-debug-ui", "Enable UI debug mode which allows to see live stylesheet updates in the UI if its used together with -custom-css-dir. It also forces some widgets to show up which are usually only visible under certain circumstances. (default: false)");
```

Comment:
```suggestion
            strUsage += HelpMessageOpt("-debug-ui", strprintf(tr("Enable UI debug mode which allows to see live stylesheet updates in the UI if its used together with -custom-css-dir. It also forces some widgets to show up which are usually only visible under certain circumstances. (default: %u)").toStdString(), false));
```

Reply:
sure? :D 

we just merged this https://github.com/dashpay/dash/commit/32fdc735039ddfc950c265f380c5a08ca9819997

Reply:
Ahh, right! ok,
```suggestion
            strUsage += HelpMessageOpt("-debug-ui", strprintf("Enable UI debug mode which allows to see live stylesheet updates in the UI if its used together with -custom-css-dir. It also forces some widgets to show up which are usually only visible under certain circumstances. (default: %u)", false));
```
then :)

Reply:
whats the point of doing this then if there is no translation? weren't translations the only reason to move out default values of the string to avoid missing translations in case of default value changes?

Reply:
Hmm, right. The default is not going to change ever here, so it's kind of pointless indeed. Ok, ignore this :)

Reply:
@UdjinM6 i just noticed that the `sprintf` with `%u` and the `false` parameter there would have made sense for another reason. The `false` doesn't make sense there in the text it should be `0` 😄